### PR TITLE
Add support for actuators and remote control using PB messages

### PR DESCRIPTION
**Pull request by ecourtois**
Add C++ classes to represent servomotors and vacuum pumps.
Initialize all actuators.
Define PB messages to send actuators state to Copilot.
Define PB messages to receive actuator commands from Copilot.
Create a thread to send actuators state.
Add uartpb handlers for actuator commands.

### CODE REVIEW

**Review by gdoffe — CHANGES_REQUESTED** [review]
Typos/comments mainly.

Good job and happy to see arms and pumps back !

File: platforms/pegasus/LxServo.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    move_stop();
+    command_ = position;
+    lx_comm_error_t ret = LX_OK;
+    ret = move_wait(position, time);
+    if (ret != LX_OK) {
+        return ret;
+    }
+    ret = move_start();
+    return ret;
+}
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply platforms/pegasus/LxServo.cpp:36]: Done

File: examples/driver_lx_servo/main.c [hunk 2]
```diff
@@ -9,7 +9,6 @@
 /* Firmware includes */
 #include "lx_servo.h"
 #include "shell.h"
-#include "shell_commands.h"
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply examples/driver_lx_servo/main.c:12]: It is related.
Added a ref to https://github.com/RIOT-OS/RIOT/pull/18152.

File: examples/driver_vacuum_pump/main.c [hunk 3]
```diff
@@ -1,6 +1,5 @@
 /* Firmware includes */
 #include "shell.h"
-#include "shell_commands.h"
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply examples/driver_vacuum_pump/main.c:3]: It is related.
Added a ref to https://github.com/RIOT-OS/RIOT/pull/18152.

File: boards/cogip-board/include/periph_conf.h [hunk 4]
```diff
@@ -105,8 +105,7 @@ static const uart_conf_t uart_config[] = {
 
 #define UART_0_ISR      (isr_usart2)
 #define UART_1_ISR      (isr_usart1)
-#define UART_2_ISR      (isr_usart3)
-#define UART_3_ISR      (isr_uart5)
+#define UART_2_ISR      (isr_uart5)
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply boards/cogip-board/include/periph_conf.h:108]: Done

File: boards/cogip-board/include/board.h [hunk 5]
```diff
@@ -26,8 +26,19 @@
 extern "C" {
 #endif
 
-/* Servomotors half duplex direction pin */
+#define GPIO_VACUUM1_ENABLE GPIO_PIN(PORT_C, 13)
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply boards/cogip-board/include/board.h:29]: Done

**Review by gdoffe — APPROVED** [review]

**Pull request merged**